nes/src/nes.cpp

```diff
@@ -44,7 +44,11 @@ class Nes::Impl {
               ppu(PpuFactory::create(&ppu_registers, ppu_mmu.get())),
               mmu(MmuFactory::create(
                       MemBankFactory::create_nes_mem_banks(ppu.get()))),
-              cpu(CpuFactory::create_mos6502(&cpu_registers, mmu.get())) {}
+              cpu(CpuFactory::create_mos6502(&cpu_registers, mmu.get())) {
+        cpu_registers.p = 0x24;
+        cpu_registers.a = cpu_registers.x = cpu_registers.y = 0x00;
```

Comment:
Weren't these 0 before since we 0-initialize the struct  when declaring the member?

Reply:
Yeah, they are. Thought it was better to make it explicit here since we modifying the other members. Can revert if you want :)

Reply:
No, this is fine. :D